packages/capabilities/test/capabilities/provider.test.js

```diff
@@ -18,7 +18,7 @@ describe('provider/add', function () {
       audience: service,
       with: account,
       nb: {
-        provider: service.did(),
+        provider: 'did:web:test.web3.storage',
```

Comment:
I'm not sure I understand rational here but I have to say I liked the version before.

Reply:
`service` can be `did:key` so didn't match the type.
Do you want me to allow `did:key` for providers?

Reply:
I think service should not be a `did:key` and we probably should fix it in cases where it is ? That said I'm ok with things as is

Reply:
ok in another PR maybe I will change the type of the `service` fixture to always only be `did:web`. I avoided that here because I didn't want to revisit/update all the tests that rely on that fixture.